client/verta/docs/tutorials/deployment/endpoint_update_config.rst

```diff
@@ -0,0 +1,57 @@
+Updating Endpoint with Config File
+==================================
```

Comment:
Please follow the template

Reply:
Not really sure what you meant... I only noticed that my "with" should probably be capitalized, based on https://github.com/VertaAI/modeldb/blob/master/client/verta/docs/tutorials/packaging/custom_model_with_dependencies.rst.

Reply:
Check the organization of the template. It should start with a simple statement of the problem being solved without getting into specifics. This example should probably be in a section like "Configuration JSON" with links on how to find out the fields available.

Reply:
Hmm the statement "Sometimes, it is more convenient to specify update configuration in a separate, human-readable file, than to have everything in a Python script or a split CLI command." was meant to be the motivation of this tutorial. 

I think the problem being solved here is described in "Verta's Client and CLI support updating Endpoint with JSON or YAML config file. This tutorial will show how it can be done, using the Client and CLI." I'll move that to the top, and separate the rest into a new section.

Reply:
For the field description, I believe Michael is working on that list. I'll add a TODO comment for updating it later.

Reply:
> I think the problem being solved here is described in "Verta's Client and CLI support updating Endpoint with JSON or YAML config file. This tutorial will show how it can be done, using the Client and CLI." I'll move that to the top, and separate the rest into a new section.

That seems more correct.